scripts/globals/abilities/stay.lua

```diff
@@ -46,9 +46,6 @@ function onUseAbility(player,target,ability,action)
         local tick = 10 - math.ceil(math.max(0, level / 20))
         --printf('tick: %d', tick)
         pet:addStatusEffectEx(EFFECT_HEALING, 0, 0, tick, 0)
-        -- I'm not sure how to set the pet animation to NONE or HEALING
-        -- and I can't find an example.  This results in the perpetuation 
-        -- of the walking/moving animation if you use stay while the pet is moving.
-        --action:animation(pet:getId(), ANIMATION_HEALING)     
+	pet:setAnimation(0)
```

Comment:
Oh that's weird..Github previews tabs differently on the commit page than it does here. It looks like a 2 space indent there, looks like 4 here. Didn't realize it was a tab. They both used to display tabs as 8.